misc/close-window.c

```diff
@@ -1,42 +0,0 @@
-#include <X11/Xutil.h>
```

Comment:
Removing files should be in separate commit (with reasoning), not with one about moving files without changes.